resilience4j-micrometer/src/main/java/io/github/resilience4j/micrometer/tagged/TagNames.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2019 Yevhenii Voievodin
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.github.resilience4j.micrometer.tagged;
+
+/** Common constants for metric binder implementations based on tags. */
+public final class TagNames {
+    public static final String NAME = "name";
+    public static final String KIND = "kind";
```

Comment:
Just out of interest. Why did you call it `kind`?

Reply:
The hardest part :fire: 

For example, for `resilience4j_circuitbreaker_calls` i read it like:
> The kind of circuit breaker calls measured by this meter

The alternatives that were also considered:
1. `characteristic`, if we name the metric `resilience4j_circuitbreaker_ring_buffer` then it would make sense, since it would be characteristic of the ring buffer, and I like this name, except the fact that it's too long/errorprone and the ring_buffer part relies on circuit breaker implementation details
2. `metric`/`measure`/`meter` wasn't sure about these ones because the actual meter name + tags is a  **metric**, and having "metric" word inside it looks weird
3. `type`/`category` basically the same as kind

Would you suggest naming it differently ?

Reply:
It would probably make sense to mirror the naming of the [Prometheus integration](http://resilience4j.github.io/resilience4j/#_prometheus_metrics_integration):
<img width="877" alt="Screen Shot 2019-03-15 at 7 34 17 PM" src="https://user-images.githubusercontent.com/131809/54467098-5aa5fd80-4759-11e9-9132-12dd6bba1e7e.png">

Reply:
Can be an option, yes. I did it this way to make everything consistent, I mean that would be weird to have this one for bulkheads
```
resilience4j_bulkhead_concurrent_calls{name="Backend1", call_result="max_allowed"}
```
But maybe it should not be that consistent.

Reply:
Let's keep the discussion in the issue https://github.com/resilience4j/resilience4j/issues/354
I already made some suggestion to rename it.
I don't want to keep the naming of the Prometheus Integration. We should improve it.